sections/plate_notation.tex

```diff
@@ -1,26 +1,29 @@
+Documents are represented as distributions of topics and topics are distributions over words.
+For each document $d \in \{1,\dots, M\}$ a document-topic distribution $\theta_d$ is sampled from a dirichlet distribution parameterized by $\alpha$, likewise for each topic $k$ a topic-word distributions $\beta_k$ is sampled from a dirichlet distribution parameterized by $\eta$.
+For each word position $n \in \{1, \dots, N_d\}$ in each document $d$, a topic $z_{d,n}$ is sampled from $\theta_d$ and then a word $w_{d,n}$ is sampled from $\beta_{z_{d,n}}$.
 The plate notation for \gls{lda} can be seen in \autoref{fig:standard_lda}.
-\todo[inline]{Some of this text about our use of standard LDA can maybe be cut down, if we have a specific LDA section and simply ref to that.}
 
 \input{figures/lda_plate}
 
 \subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
-\citet{author_topic_2012} describes the author-topic model where for each document $d$, they assign a vector of authors $a_d$ from a set of authors $A$, and for each word draw an author $x$ from this vector.
-In our category-topic model and our own author-topic model, we use a similar approach to the author-topic model from \citeauthor{author_topic_2012}, where each document $d$ is associated with one category $c_d$ from a set of categories $C$ or an author $a_d$ from a set of authors $A$.
+We model both of the meta-data fields 'Author' and 'Category' similarly to the model presented by \citet{author_topic_2012}.
+In this model, there are no document-topic distributions $\theta$.
+Instead, each author and category has its own topic distribution.
+This is based on the assumption that authors prefer to write about specific topics, and that categories of the articles were chosen based on the content of the finished article or that local newspapers have their own unique topic preferences.
 
-Unless otherwise states, future references are to our implementation of this model, rather than the model by \citet{author_topic_2012}.
+Our topic models are \glspl{um}, meaning that topics are only word distributions and are chosen based on the data of the documents, rather than \glspl{dm} where topics are fitted to have an influence on both word distributions and other metadata.
+
+\citet{author_topic_2012} describes the author-topic model where for each document $d$, they assign a vector of authors $a_d$ from a set of authors $A$, and for each word draw an author $x$ from this vector.
+However, for our category-topic model and our own author-topic model, each document $d$ is associated with one category $c_d$ from a set of categories $C$ and one author $a_d$ from a set of authors $A$, rather than a vector.
+This is due to our dataset never having more than one author or category for each document.
+Unless otherwise states, future references are to our implementation of this model, rather than the model presented by \citet{author_topic_2012}.
```

Comment:
```suggestion
Unless otherwise stated, future mentions of the author-topic model are to our implementation of this model, rather than the model presented by \citet{author_topic_2012}.
```